border-radius-previewer/src/App.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+
+const App: React.FC = () => {
+  const [upperLeft, setUpperLeft] = useState<string>("0px ");
+  const [upperRight, setUpperRight] = useState<string>("0px ");
+  const [lowerLeft, setLowerLeft] = useState<string>("0px ");
+  const [lowerRight, setLowerRight] = useState<string>("0px ");
```

Comment:
```suggestion
  const [upperLeft, setUpperLeft] = useState(0);
  const [upperRight, setUpperRight] = useState(0);
  const [lowerLeft, setLowerLeft] = useState(0);
  const [lowerRight, setLowerRight] = useState(0);
```
データと書式は分離したほうがいいので、pxとかスペースとかをstateに含めないほうがいいかな